com.civicrm.hrjobroles/CRM/Hrjobroles/DAO/HrJobRoles.php

```diff
@@ -508,25 +508,27 @@ static function &fields()
         ) ,
         'hrjc_role_start_date' => array(
           'name' => 'start_date',
-          'type' => CRM_Utils_Type::T_DATE | CRM_Utils_Type::T_TIME,
+          'type' => CRM_Utils_Type::T_DATE + CRM_Utils_Type::T_TIME,
+          'default' => 'NULL',
```

Comment:
well, since the default was 0 then it fine for now to change it to null , it could be solved  later in another ticket ..